apps/election-manager/src/config/types.ts

```diff
@@ -103,6 +103,21 @@ export enum TallyCategory {
 export interface FullElectionTally {
   readonly overallTally: Tally
   readonly resultsByCategory: ReadonlyMap<TallyCategory, Dictionary<Tally>>
+  readonly externalTally?: FullElectionExternalTally
+}
+
+export interface ExternalTally {
+  readonly contestTallies: Dictionary<ContestTally>
+  readonly numberOfBallotsCounted: number
+}
+
+export interface FullElectionExternalTally {
+  readonly overallTally: ExternalTally
+  readonly resultsByCategory: ReadonlyMap<
+    TallyCategory,
+    Dictionary<ExternalTally>
+  >
+  readonly fileContent: string
```

Comment:
This is stored so that in the combination step the original file can be sent back to the exisiting /combine endpoint.

Reply:
could we do away with the `/combine` endpoint and just do a simple SEMS export of our combined tally?

Reply:
We could but it would add a considerable amount of work. The endpoint right now to convert our results to the SEMs results just takes the raw CVRs file, not the computed tallys, and recomputes all the tallies, which allows us to do things like count undervotes slightly differently for SEMs. I'm not sure we would want to rearchitect all of that...

Reply:
Apologies if this was clear to anyone else, but does this mean we have two different systems doing the conversion and/or tallying? How are we verifying that they produce the same results?

Reply:
We only have 1 system for converting (although VotingWorks -> SEMs lives in module-converter-sems and SEMs -> VotingWorks lives in election manager), but we do have 2 systems for tallying (which was true prior to any work I've done related to this). Module-converter-sems takes the CVRs files and tallies thoses / converts the tallies to the SEMs format, and election-manager takes CVRs file and tallies them for our internal tally reports. Right now what SEMs considers the right value for an undervote is different then what the VVSG 2.0 standard says an undervote should be so there are a few details like that where it kind of makes sense that it is a different system calculating it. But I agree that in general its a fragile setup. Filed https://zube.io/votingworks/vxsuite/c/3028 to track if we want to refactor this in the future. 

The /combine endpoint doesn't really do any converting/interpreting FWIW it just literally sums identical rows in the CSVs.